l10n/en/firefox/installer-help.ftl

```diff
@@ -0,0 +1,31 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/installer-help/
+
+## Required
+installer-help-page-title = Your download was interrupted
+installer-help-main-heading = Hmm… your download was interrupted
```

Comment:
Is this really simple English, or should it move to an en-US file?

Reply:
Good catch - yeah let's move this to an en-US file.